README.md

```diff
@@ -28,6 +34,6 @@ Quick version:
 - Build the image and start the container with `docker-compose build && docker-compose up`.
 - Enter the docker container with `docker-compose run web bash`.
 - Inside the docker container run `python manage.py createsuperuser` and create a user.
-- Login at `localhost:8000/admin` with the created user.
+- Login at `localhost:5000/admin` with the created user.
```

Comment:
I think this is for GnosisDB Django Admin. Can you check?

Reply:
You are right Andre, I'm pushing the fix